### DISCUSSION TITLE
Case-insensitive Sortierung bei Inventarnummer

### DISCUSSION THREAD

**Discussion by steffifitz** [Samedis-care/samedis-care-community#84] [Shipped - Live on production]
Aktuell ist die Sortierung bei den Inventarnummern case-sensitiv. Finde ich nicht so praktisch, da nicht intuitiv. Beispiel siehe Anhang.
![Sortierung_Inventarnummer](https://github.com/Samedis-care/samedis-care-community/assets/140176532/f679f5a3-2e0a-4286-9621-7c9f05d5c282)

**Comment by neongrau** [comment]
Hallo Frau Fitz,

das dürfte seit letztem Freitag behoben sein.

Viele Grüße
Ralf Vitasek
